app/modules/ads/slots.js

```diff
@@ -244,6 +257,27 @@ export default {
         },
         trackingKey: 'video',
       },
+      incontent_native: {
+        aboveTheFold: true,
+        firstCall: false,
+        defaultSizes: ['fluid'],
+        adProduct: 'incontent_native',
+        slotNameSuffix: '',
+        nonUapSlot: true,
+        group: 'NATIVE',
+        slotShortcut: 'n',
+        lowerSlotName: 'incontent_native',
+        sizes: [
```

Comment:
We can define empty sizes (`sizes: []`) if we don't need to restrict them by viewport. `defaultSizes` is enough